### ISSUE TITLE
@1 - @3 index export issue

### ISSUE THREAD

**Issue by Samgoldman123** [tudou527/marketch#78]
Amazing plugin, freaking awesome job can't applaud this enough.

I have been using the plugin for over 5 months now with no issues, upon either a plugin update or sketch3 update my index inspect (top right panel) now only allows me to select @1 or @3... no @2. 
Also when i download the @1 or @3 the sizing is incorrect, if i select an image @1 and download it the sizing is relevant to about @.5   its weird.. 
I have uninstalled/reinstalled, restarted Sketch tried it on multiple files all with the same issue..

Sketch
Version 3.5.1 (25234)

Marketch
1.0.13 i

Any help would be amazing.

**Comment by jand000** [comment]
Just to confirm: I have the same issue here. (Sketch 3.5.1, Marketch 1.0.13, OS X 10.11.3)

**Comment by tudou527** [comment]
do you design as 1x? marketch think all the design is 2x as default. so 0.5x is the real size, and 3x also base on the real size. if you desigin on 1x, this will cause problems.

**Comment by Samgoldman123** [comment]
This is new, of course i understand the process of designing at 2x as a foundation but have never had to as Marketch in the past gave me the option for X1 X2 X.5  so I had no issues with designing at baseline. 

So basically what your saying is there will be no fix for this and I should change my design approach? 
So what about other design elements like type styling and Div heights? The developers inspecting these have to half the inspected Marketch dimensions because I design @2 as a baseline...??

Prior to the last plugin update everything was working like a dream, X1 X2 X.5 options = great.
Appreciate your comments/feedback, great work once again.

**Comment by nClear** [comment]
I've been using this plugin for months also for design web-pages (1x, for sure). And its was great to use this plugin for export pages for people who making code but don't have Sketch. 
Now this plugin useless for this situation (

**Comment by Samgoldman123** [comment]
Legends!
I have to confess, as I had developers waiting and I had to try and find an alternative, I ended up trying Avocode and it was not the best exercise.. No @2@3 options...
 Your product is gold, I design UI for app and web and your plugin bridges a massive gap between design and dev(and I now can stick to basline design and not @2 base). Even if this were to go mainstream and sold as a product it would be worth it, maybe some more love placed into the UI design of it but functionally = gold. All of the developers I work with LOVE it, great job fellas, be proud.
